src/modules/twap/hooks/useTwapOrdersTradeableMulticall.ts

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from 'react'
+
+import { ListenerOptionsWithGas } from '@uniswap/redux-multicall'
+
+import { ComposableCoW, GPv2Order } from 'abis/types/ComposableCoW'
+import { useSingleContractMultipleData } from 'lib/hooks/multicall'
+
+import { ConditionalOrderParams } from '../types'
+
+const DEFAULT_LISTENER_OPTIONS: ListenerOptionsWithGas = { gasRequired: 185_000, blocksPerFetch: 5 }
+
+export type TradeableOrderWithSignature =
+  | {
+      order: GPv2Order.DataStructOutput
+      signature: string
+    }
+  | undefined
+
+export function useTwapOrdersTradeableMulticall(
+  safeAddress: string,
+  composableCowContract: ComposableCoW,
+  conditionalOrderParams: ConditionalOrderParams[]
+): TradeableOrderWithSignature[] {
```

Comment:
This hook is pretty generic, is not related to TWAP afait. It feels like we could use a "composableCoW" module or something